app/src/internal/java/com/kickstarter/ui/activities/DesignSystemActivity.kt

```diff
@@ -116,6 +116,25 @@ fun DesignSystemViewPreview() {
     }
 }
 
+@Preview(name = "Light", uiMode = Configuration.UI_MODE_NIGHT_NO, showSystemUi = true)
+@Preview(name = "Dark", uiMode = Configuration.UI_MODE_NIGHT_YES, showSystemUi = true)
+@Composable
+fun DesignSystemButtonsV2Preview() {
+    val currentTheme = isSystemInDarkTheme()
+    var darkMode = remember { mutableStateOf(currentTheme) }
```

Comment:
wondering why we need lines 123,124 and 126. Just with the preview annotations seems to work fine but not sure if Im missing something
<img width="1246" alt="Screenshot 2025-05-08 at 12 46 46 PM" src="https://github.com/user-attachments/assets/06ec3593-ebd7-42ee-a605-8c4a3daad425" />

Reply:
I was having trouble getting the dark mode preview to work properly for these buttons, i think this is a remnant of me fiddling around with it. I can remove!